### ISSUE TITLE
NoSQL / MongoDB Level not working after services separated in docker

### ISSUE THREAD

**Issue by ismisepaul** [OWASP/SecurityShepherd#373]
When accessing the NoSQL level the following error is observed;

```bash
MongoDb Error caught - com.mongodb.MongoTimeoutException: Timed out after 10000 ms while waiting to connect. Client view of cluster state is {type=Unknown, servers=[{address=127.0.0.1:27017, type=Unknown, state=Connecting, exception={com.mongodb.MongoException$Network: Exception opening the socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]
```

The mongoSchema.js should be reviewed and possibly a properties file created to connect to mongo.

**Comment by markdenihan** [comment]
Hey @ismisepaul ,
Is there any known work around for this problem?
Perhaps this should be disabled by default until no workaround is required?
This is blocking #393

**Comment by markdenihan** [comment]
I'll make an issue to disable this level in Docker deployments until this is fixed in 3.2

**Comment by ismisepaul** [comment]
Work has been done before #294
